src/core/server/saved_objects/migrations/actions/integration_tests/actions.test.ts

```diff
@@ -321,8 +321,12 @@ describe('migration actions', () => {
   });
 
   describe('waitForIndexStatusYellow', () => {
-    afterAll(async () => {
-      await client.indices.delete({ index: 'red_then_yellow_index' });
+    afterEach(async () => {
+      try {
```

Comment:
We reuse the same index for both tests in the `describe` block. I added the `try/catch` as a safety net for if we one day skip one of the tests.